src/linter/xmlTemplate/linter.ts

```diff
@@ -4,12 +4,14 @@ import transpileXml from "./transpiler.js";
 import {LinterParameters} from "../LinterContext.js";
 import ControllerByIdInfo from "./ControllerByIdInfo.js";
 import {ControllerByIdDtsGenerator} from "./generator/ControllerByIdDtsGenerator.js";
+import {decodedMap, DecodedSourceMap, SourceMapInput, TraceMap} from "@jridgewell/trace-mapping";
 
 // For usage in TypeLinter to write the file as part of the internal WRITE_TRANSFORMED_SOURCES debug mode
 export const CONTROLLER_BY_ID_DTS_PATH = "/types/@ui5/linter/virtual/ControllerById.d.ts";
 
-export default async function lintXml({filePathsWorkspace, workspace, context}: LinterParameters) {
-	const xmlResources = await filePathsWorkspace.byGlob("**/{*.view.xml,*.fragment.xml}");
+export default async function lintXml({filePathsWorkspace, workspace, context, altGlob}: LinterParameters) {
```

Comment:
As this function accepts the common `LinterParameters`, but we only want to implement handling of `altGlob` for the lintXml, I'd suggest to add that parameter only here, not in the general `LinterParameters`. Otherwise one might try to use the parameter with the other functions.

Reply:
That was exactly my point. It allows extensibility in the future for the other linters.
I'll adjust it as per your comment